src/pages/en/guides/deploy/github.md

```diff
@@ -73,9 +73,15 @@ You can deploy an Astro site to GitHub Pages by using [GitHub Actions](https://g
     :::tip
     See [the GitHub Pages Action documentation](https://github.com/marketplace/actions/github-pages-action) for different ways you can configure the final “Deploy to GitHub Pages” step.
     :::
+3. (Optionally) Set up a custom domain by adding the following `./public/CNAME` file to your project: 
+```
+sub.mydomain.com
+````
+`actions-gh-pages` will detect it in `./dist` after build, and will replace `user.github.io` with the correct custom destination domain for your deployment environment. Don't forget to also [configure DNS for your domain provider](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain).     
+
```

Comment:
We should indent these items by four spaces to make sure they get nested correctly under item 3. I’ve also added a filename to the example CNAME code block.

````suggestion

    ```txt title="public/CNAME"
    sub.mydomain.com
    ```

    `actions-gh-pages` will detect it in `./dist` after build, and will replace `user.github.io` with the correct custom destination domain for your deployment environment. Don't forget to also [configure DNS for your domain provider](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain).     

````